site/content/contribute/webapp/using-i18n-extract.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Using make i18n-extract"
+date: 2020-04-20T08:31:17-04:00
+weight: 7
+subsection: Web App
+---
+
+`make i18n-extract` is a command used for localization. It allows you to validate that your strings have been successfully extracted from your source code before you continue.
+
+This page demonstrates how to review your results and to verify if your extraction was successful or not. If the extraction was not successful this page also provides a workaround to correct for this.
+
+NOTE: These steps apply for the `mattermost-webapp` repository only.
+
+1. After you execute `make i18n-extract` you will need to review the results and validate that the strings were either added or removed in the `i18n/en.json` file. 
+2. Run  `git diff` and determine if your strings were added or removed in the `i18n/en.json` file correctly. If this was a successful extraction you will have output similar to below:
+
+<img src="/img/i18n-extract-1.jpg" width="445" />
```

Comment:
Resubmitting from https://github.com/mattermost/mattermost-developer-documentation/pull/541 where these were images - but as discussed with submitter, an improvement would be to include these as code samples.

Editor note: Unable to render these images in GitHub preview mode, not sure if I've made an error.

Reply:
I think you embed the images with `![image](/img/i18n-extract-1.jpg)`. Might we locate these files closer to the source in question, vs. being global statics? But I 100% agree that this would be best as a code sample instead with the `diff` formatting style.

Reply:
Thanks! That helped and I specified the exact source of the image file